html/images/os/fs.svg

```diff
@@ -0,0 +1,140 @@
+<?xml version="1.0" encoding="utf-8"?>

```

Comment:
This needs to be optimized, I usually just paste it into https://jakearchibald.github.io/svgomg/